social_marketing/overview/manage_social.rst

```diff
@@ -0,0 +1,79 @@
+============================================
+Manage Social Media Pages and Create Content
+============================================
+
+Once you are present on social media platforms, it is important to keep them up-to-date. However,
+that might feel like a full time job. Having a solution that allows you to create, schedule, keep
+track, engage with your audience and measure results saves you time and helps you to successfully
+execute your online strategy, from posts to results.
+
+
+Add accounts and create your feed
+=================================
+
+Click on *Add Stream* and grant the required permissions to add your accounts.
+
+.. image:: media/add_streams.png
+   :align: center
+   :alt: Click on add streams to add accounts in Odoo Social Marketing
+
+.. note::
+   You can link one Facebook page where you are the admin.
+
+| You will be redirected to your *Feed* and a column will be automatically added with the
+  publications.
+| You can then add new accounts and/or streams, such as youtFacebook mentions for example, and
+  customize your Kanban view.
+
+.. image:: media/feed.png
+   :align: center
+   :alt: See all streams in the feed in Odoo Social Marketing
+
+Adding social media accounts to your feed will also link KPIs. To get redirected to the statistics
+page of the social account click on *Insights*.
```

Comment:
```suggestion
page of the social account, click on *Insights*.
```

Reply:
This comma, here, is quite essential.
I know that sometimes I suggest commas that are not that important, but this one is. ^^